common-arcana.lic

```diff
@@ -236,4 +236,57 @@ module DRCA
       return parse_mana_message(mana_msg)
     end
   end
+
+  def check_osrel(osrel_no_harness, osrel_amount)
+    return unless DRSpells.active_spells['Osrel Meraud'] && DRSpells.active_spells['Osrel Meraud'] < 90
+    infuse_om(!osrel_no_harness, osrel_amount)
+  end
+
+  def do_waggle_set(settings, spells)
+    check_osrel(settings.osrel_no_harness, settings.osrel_amount)
```

Comment:
This was the previous behaviour in `buff.lic`. I'm a little hesitant to change that without a cleric to test on.